docs/streams/upgrade-guide.html

```diff
@@ -77,7 +77,7 @@ <h3><a id="streams_api_changes_250" href="#streams_api_changes_250">Streams API
         We add a new <code>cogroup()</code> operator (via <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-150+-+Kafka-Streams+Cogroup">KIP-150</a>>)
         that allows to aggregate multiple streams in a single operation.
         Cogrouped streams can also be windowed before they are aggregated.
-        We refer to the <a href="/{{version}}/documentation/streams/developer-guide/dsl-api.html">developer guide</a> for more details.
+        Please refer to the <a href="/{{version}}/documentation/streams/developer-guide/dsl-api.html">developer guide</a> for more details.
```

Comment:
Rephrase this sentence. 
Before: `We refer to the developer guide for more details.`
After: `Please refer to the developer guide for more details.`